tests/testutils/test_utils.go

```diff
@@ -404,3 +406,48 @@ func compress(content string) []byte {
 	_ = bw.Close()
 	return buf.Bytes()
 }
+
+func WithManager(t *testing.T, inst *instance.Instance) (err error, shouldRemoveUUID bool) {
+	if inst.UUID == "" {
+		uuid, err := uuid.NewV4()
+		if err != nil {
+			return fmt.Errorf("Could not enable test instance manager: %w", err), false
```

Comment:
You can use `require.NoError(t, err, "Could not enable test instance manager")` here instead of returning an error

Reply:
Ah yes, I didn't think about changing the behavior after passing `t`.